tests/test_e2e.py

```diff
@@ -95,8 +98,7 @@ def client_select_directory(port: int) -> httpx.Response:
 
     assert cli_result.exit_code == 0
     webbrowser_open_mock.assert_called_once()
-    output_file = tmp_path / "REDACTED_test_image.tif"
-    assert output_file.exists()
+    output_file = tmp_path / "Redacted_2023-05-12_12-12-53/REDACTED_test_image.tif"
```

Comment:
These need to be separate components.

Reply:
fixed in a0c2bf04704b088fb3ba1cfdeb17ff930b8983bf